eng/pipelines/runtime.yml

```diff
@@ -326,7 +325,7 @@ jobs:
       testGroup: innerloop
       nameSuffix: AllSubsets_Mono
       buildArgs: -s mono+libs+installer+libs.tests -c $(_BuildConfig) /p:ArchiveTests=true
-      timeoutInMinutes: 120
+      timeoutInMinutes: 240
```

Comment:
Is this because building the android tests takes a long time, or to actually running them?

Reply:
At the time, there was only one booted Helix VM running Android tests, which take a combined ~3 hours on one host